crypto/signer_test.go

```diff
@@ -65,12 +66,18 @@ func TestSigner(t *testing.T) {
 	}{
 		{message: []byte("message")},
 	} {
-		signature, err := signer.Sign(test.message)
+		sig, err := signer.Sign(test.message)
+		if got, want := sig.HashAlgorithm, sigpb.DigitallySigned_SHA256; got != want {
+			t.Fatalf("Hash alg incorrect, got %v expected %d", got, want)
+		}
+		if got, want := sig.SignatureAlgorithm, sigpb.DigitallySigned_ECDSA; got != want {
+			t.Fatalf("Sig alg incorrect, got %v expected %v", got, want)
+		}
 		if err != nil {
```

Comment:
Move `if err != nil` to line 70, add a `continue` after the `t.Errorf("Failed to sign ...")` below. We shouldn't assume `sig != nil` if `err != nil`.